tests/postgres/range_test.go

```diff
@@ -0,0 +1,419 @@
+package postgres
+
+import (
+	"github.com/go-jet/jet/v2/internal/testutils"
+	"github.com/go-jet/jet/v2/qrm"
+	"github.com/google/go-cmp/cmp"
+	"github.com/jackc/pgtype"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"testing"
+	"time"
+
+	. "github.com/go-jet/jet/v2/postgres"
+	"github.com/go-jet/jet/v2/tests/.gentestdata/jetdb/test_sample/model"
+	. "github.com/go-jet/jet/v2/tests/.gentestdata/jetdb/test_sample/table"
+)
+
+func TestRangeTable_DateContainsSingle(t *testing.T) {
+	skipForCockroachDB(t)
+	expectedSQL := `
+SELECT DISTINCT sample_ranges.date_range AS "sample_ranges.date_range",
+     sample_ranges.timestamp_range AS "sample_ranges.timestamp_range",
+     sample_ranges.timestampz_range AS "sample_ranges.timestampz_range",
+     sample_ranges.int4_range AS "sample_ranges.int4_range",
+     sample_ranges.int8_range AS "sample_ranges.int8_range",
+     sample_ranges.num_range AS "sample_ranges.num_range"
+FROM test_sample.sample_ranges
+WHERE sample_ranges.date_range @> '2023-12-12'::date;
+`
+
+	query := SELECT(SampleRanges.AllColumns).
```

Comment:
Could you include also here calls to UPPER_BOUND, LOWER_BOUND, as well as range constructors and infinities?